scripts/update_PR_comment.sh

```diff
@@ -26,11 +26,11 @@ if [[ -z $PR ]]; then
   echo "Usage: <new_message> <beginning of message> <pr_number>"
```

Comment:
```suggestion
  echo "Usage: <new_message_file> <beginning of message> <pr_number>"
```

Reply:
Thanks!  I took the opportunity to also change the name of the other variable to `beginning_of_old_message` to clarify the distinction between what you expect to already be there and what is the new part that gets added.